lib/utils/isHexColor.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const HEX = /^#[\da-z]+$/i;
```

Comment:
suggestion
enhancement

Maybe we could restrict the accepted lengths to 3, 6 and 8?

Reply:
Good idea. Import `isValidHex` instead of using the regex.